### DISCUSSION TITLE
Does the dgs framework support GraphQL Schema Stitching (type merging)?

### DISCUSSION THREAD

**Discussion by mrtinkz** [Netflix/dgs-framework#398] [Q&A]
Hi, does the dgs framework support GraphQL Schema Stitching? I am referring to new type merging support added to the schema stitching. If so are there any examples on how we could do that? Below is link to what I am referring to in grapql tools, thanks

[GraphQL Schema Stitching Support](https://www.graphql-tools.com/docs/schema-stitching/)

**Comment by paulbakker** [comment]
Since we're using Federation, we don't do schema stitching. I'm not familiar enough with schema stitching to give any pointers about if/how that's supported or not.

It would be interesting for someone with schema stitching experience to explore this.
